lib/assessment.rb

```diff
@@ -43,6 +44,11 @@ def transform(words)
   # replacing the trailing newline character with a
   # space and return the result.
   def read_file(filename)
-    # Your code goes here
+    File.open(ARGV[0]) do |file|
+      lwc = file.each_with_object(Hash.new(0)) do |line, memo|
+      memo[:lines] += 1
+      end
+    end
   end
-end
+#I know the keyword chomp removes linebreaks
+#and needs to be implemented somewhere ( maybe with >>)
```

Comment:
Yeah the functionality's missing. Here's Antony's solution.
def read_file(filename)
```ruby
File.open(filename).each_with_object('') do |line, memo|
   memo << line.chomp + ' '
end.strip
```